app/views/home.pug

```diff
@@ -7,8 +7,10 @@ block content
 			.col-md-6.col-md-offset-3
 				.card
 					form(action="/login", method="POST")
-						if failedLogIn
+						if failedLogin
 							.alert.alert-danger Your email or password was incorrect
+						if failedOneTimeLogin
+							.alert.alert-danger Your login link has expired
```

Comment:
I think it's always a good thing to advise the user what to do next when there's an error. So maybe "Your login link has expired. Please request another one." (ideally linking back to the place they can do that).